src/analyzer/statistics_calculator.py

```
@@ -0,0 +1,485 @@
+
+    # 如果没有配置词组，创建一个包含所有新闻的虚拟词组
+    if not word_groups:
+        print("频率词配置为空，将显示所有新闻")
+        word_groups = [{"required": [], "normal": [], "group_key": "全部新闻"}]
+        filter_words = []  # 清空过滤词，显示所有新闻
+
+    is_first_today = False  # 这里需要从外部获取配置
+    try:
+        from ..utils.time_utils import is_first_crawl_today
+        is_first_today = is_first_crawl_today()
+    except:
```

Ruff (0.14.8)

Do not use bare `except`

(E722)